storage/innobase/xtrabackup/doc/source/installation/docker.rst

```diff
@@ -0,0 +1,124 @@
+.. _pxc.installing/docker.running:
+
+Running |percona-xtrabackup| in a |docker| container
+********************************************************************************
+
+You may run |percona-xtrabackup| in a |docker| container without
+having to install it. All required libraries come already installed in
+the container.
+
+Being a lightweight execution environment, |docker| containers enable creating
+configurations where each program runs in a separate container. You may run
+|percona-server| in one container and |percona-xtrabackup| in another.
+
+You create a new |docker| container based on a |docker| image, which works as a template for
+newly created containers: perconalab/percona-xtrabackup. |docker| images for
```

Comment:
As it is already tested we need to put image under percona account on docker hub and do not tell anything regarding perconalab